src/main/java/com/Saurabh/GitHubAiAPI/StringManipulation.java

```diff
@@ -45,4 +53,11 @@ public static String findVowels(String input) {
         }
         return vowels.toString();
     }
+
+    // Helper method to check if a string is a palindrome
+    public static boolean isPalindrome(String input) {
+        String cleanInput = input.replaceAll("\\s+", "").toLowerCase(); // remove spaces and convert to lower case
+        String reversedString = new StringBuilder(cleanInput).reverse().toString();
+        return cleanInput.equals(reversedString);
```

Comment:
The isPalindrome method currently removes spaces from the input string. This might not be the desired behavior in all cases. Consider adding a parameter to control whether spaces should be removed or not.